src/Elements/utilities.jl

```diff
@@ -0,0 +1,35 @@
+["Helper functions."]
+
+"""
+Find the final configuration fo node `n`.
+
+Search begins with the node `n` itself followed by any parent `Node` configuration setting.
+"""
+function findconfig(n::Node)
+    local config
+    # Stage 1: nodes's config.
+    config = n.data
+
+    # Stage 2: Parent's config.
+    while isdefined(n, :parent)
+        n = n.parent
+        update!(config, n.data)
+    end
+    config
+end
```

Comment:
I thought about that: but my conclusion was that if one looks for a couple of symbols it has always to rerun all.

* I had a version with the symbol lookup.

  `getconfig(n::Node, k:Symbol)` but was unsure what to do if the key is not found. If we throw an error we always have to account for that in the code?

 I thought maybe return `nothing` and in the code? Or better use `try` ?

  ```
  f = getconfig(docs, :filter)
  f != nothing && ....
  ```

* > Basically, once created the document(...) object shouldn't be changed

  Do avoid re-run should we catch the final config when created the document(...) ?